openapi.yaml

```diff
@@ -23,6 +23,7 @@ paths:
   /games:
     post:
       summary: Create a poker game
+      description: The player created with memberId will have initialized value in terms of status(checked) and playerBet(0).
```

Comment:
Could you rewrite this as `players`, `memberIds`, and `values` so that it's clear that this will happen for all players? Also, I think `status` should be `playerStatus`